### PR TITLE
Add warning about Windows line-endings issues to docs

### PR DESCRIPTION
**Pull request by haraldreingruber**
Hi @paroga,

I've updated the docs with a warning about line-ending issues when using the Docker environment on Windows.

Should we add the section also to the non-docker development setup instructions? I guess it is possible to run the rails server on windows without docker as well?

Thanks, Harald

### CODE REVIEW

**Comment by paroga** [comment]
To he honest: I don't use windows and therefore don't know where the problems are. So it's completely up to you, to decide where some documentation makes sense.

**Comment by haraldreingruber** [comment]
@paroga I've updated my PR.
Let me know if you have any feedback on the updated docs.

Thanks,
Harald

File: doc/SETUP_DEVELOPMENT.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ If instead you just want to run Foodsoft without changing its code, please refer
    This brings up the bleeding-edge development version, which might contain some
    unfinished parts. If you want to be safe, choose the last release:
    `git checkout $(git tag -l | grep ^v | sort -rn | head -n1)`
+   
+   *Note:* When developing on Windows you might run into issues with shell scripts because of Git auto-crlf. Have a look how to avoid that in the [Docker Development Setup](./SETUP_DEVELOPMENT_DOCKER.md#prerequisites-windows-only) instructions.
```
Comment (paroga) [inline doc/SETUP_DEVELOPMENT.md]: please keep the maximum length in this fil

File: doc/SETUP_DEVELOPMENT.md [hunk 2]
```diff
@@ -182,3 +184,4 @@ within a docker image. While the default [`Dockerfile`](../Dockerfile) is setup
 use docker-compose (using [`docker-compose-dev.yml`](../docker-compose-dev.yml)) to
 setup the whole stack at once.
 
+See [Setup Development Docker](./SETUP_DEVELOPMENT_DOCKER.md) for a detailed description.
```
Comment (paroga) [inline doc/SETUP_DEVELOPMENT.md]: please add a linefeed at the end of the file

**Comment by paroga** [comment]
pushed manually as 9897c1056fbdc047243ea1feb8f6a57be79ea3a2. please remove tailing whitespace in the future. there are many text editors which will do that for you during save...
thx for the PR!

**Pull request closed without merging**